docs/specs/wire-protocol.md

```diff
@@ -61,12 +61,45 @@ Noise_pqXX_Xwing_ChaChaPoly_BLAKE2b
 ```
 
 The protocol string is a very condensed description of our protocol.
-We use the pqXX two way Noise pattern which is described as follows:
+We use the pqXX two way Noise pattern which is described as follows
+using the algebraic token system from the PQ Noise paper that very
+closely resembles the Noise algebraic token system:
 
 ```
-pqXX: -> e <- ekem, s -> skem, s <- skem
+-> e
+<- ekem, s
+-> skem, s
+<- skem
 ```
 
+However let us remark that usually `e` is first sent over by the
+client. However in our case the serer is the Noise protocol initiator
+after the connection is made via the client initiating the connection.
+To be clear, clients connec to servers as usually, and then the client
+waits for a message, while the server sends the first PQ Noise
+message. The reason for this modification is so that the server authenticates itself to the client first
+just like classical Noise XX pattern.
```

Comment:
It's referring to the modification being made in this PR.

Reply:
> And, in any case, why is it that pqXX needs to do it differently?

Are you asking me why is it that servers should authenticate themselves to clients first? It's because we are building an anonymous communication network and it's just completely inappropriate to have clients authenticate themselves to servers first. you dig?